tests/backend/specs/api/tidy.js

```diff
@@ -1,27 +1,16 @@
-const assert = require('assert');
-os = require('os'),
-fs = require('fs'),
-path = require('path'),
-TidyHtml = null,
-Settings = null;
-
-const npm = require('../../../../src/node_modules/npm/lib/npm.js');
+const assert = require('assert').strict;
+const os = require('os');
+const fs = require('fs');
+const path = require('path');
+const TidyHtml = require('../../../../src/node/utils/TidyHtml');
+const Settings = require('../../../../src/node/utils/Settings');
 const nodeify = require('../../../../src/node_modules/nodeify');
```

Comment:
These modules are in a separate package so they should be referenced like this:

```suggestion
const TidyHtml = require('ep_etherpad-lite/node/utils/TidyHtml');
const Settings = require('ep_etherpad-lite/node/utils/Settings');
const nodeify = require('ep_etherpad-lite/node_modules/nodeify');
```